src/api/app.js

```diff
@@ -10,15 +10,20 @@ module.exports.manifest = {
   authoriseContainer: 'promise',
   authoriseShareMd: 'promise',
   webFetch: 'promise',
-  isRegistered: 'promise',
   networkState: 'promise',
+  isNetStateInit: 'sync',
+  isNetStateConnected: 'sync',
+  isNetStateDisconnected: 'sync',
+  clearObjectCache: 'promise',
+  isMockBuild: 'promise',
```

Comment:
These new functions need to also be listed in the `documentation.yml` file

Reply:
oooooh, that's why I can't get them to show up. thank you :-D